### PR TITLE
[receiver/mysql] client, convert NULL to int64

### PR DESCRIPTION
**Pull request by knarfli**
**Description:** <Describe what has changed.>
bug fix, receiver/mysql client.go raise error when the TABLE_ROWS column is NULL,
fix it in the SQL query, convert the NULL to 0.

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34195
**Testing:** <Describe what testing was performed and which tests were added.>
run command "make test" in the receiver/mysqlreceiver, all the test case passed.
**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by djaglowski — COMMENTED** [review]
LGTM. Please sign the CLA

**Comment by knarfli** [comment]
Hi @djaglowski, change log added

File: receiver/mysqlreceiver/client.go [hunk 1]
```diff
@@ -243,8 +243,11 @@ func (c *mySQLClient) getInnodbStats() (map[string]string, error) {
 
 // getTableStats queries the db for information_schema table size metrics.
 func (c *mySQLClient) getTableStats() ([]TableStats, error) {
-	query := "SELECT TABLE_SCHEMA, TABLE_NAME, TABLE_ROWS, " +
-		"AVG_ROW_LENGTH, DATA_LENGTH, INDEX_LENGTH " +
+	query := "SELECT TABLE_SCHEMA, TABLE_NAME, " +
+	    "COALESCE(TABLE_ROWS, 0) as TABLE_ROWS, " +
```
Comment (djaglowski) [inline receiver/mysqlreceiver/client.go:247]:
```suggestion
		"COALESCE(TABLE_ROWS, 0) as TABLE_ROWS, " +
```

**Review by djaglowski — APPROVED** [review]
LGTM, but failing lint check on `client.go`. I think it's just this indent.

**Comment by knarfli** [comment]
Hi @djaglowski, is there anything else we need to do, can we merge the PR?

**Pull request merged**